app/public/templates/scene.html

```diff
@@ -125,12 +125,17 @@
           {% if object["is_interactable"] %}
             class="link"
             animation__onclick="property: position; dur: 0; startEvents: click"
-            animation-on-click-run='blackboardData: {% raw json.dumps(object["animations_json"]["blackboardData"]) %}'
+            animation-on-click-run='blackboardData: {{ json_encode(object["animations_json"]["blackboardData"]) }}'
           {% end %}
         ></a-gltf-model>
         {% end %}
       </a-scene>
       <script>
+        function htmlDecode(input) {
```

Comment:
I know I wasn't set as a reviewer, but this function isn't being used? Also a console log

Reply:
Oh you read my mind

Reply:
Already removed it